src/hydro/hydro_cuda_tests.cu

```diff
@@ -0,0 +1,90 @@
+/*!
+* \file hydro_cuda_tests.cu
+* \author Evan Schneider (evs34@pitt.edu)
+* \brief Test the code units within hydro_cuda.cu
+*
+*/
+
+// STL Includes
+#include <iostream>
+#include <vector>
+#include <string>
+#include <stdlib.h>
+
+// External Includes
+#include <gtest/gtest.h>    // Include GoogleTest and related libraries/headers
+
+// Local Includes
+#include "../global/global.h"
+#include "../global/global_cuda.h"
+#include "../utils/gpu.hpp"
+#include "../utils/testing_utilities.h"
+#include "../hydro/hydro_cuda.h"   // Include code to test
+
+#if defined(CUDA)
+
+
+TEST(tHYDROCalcDt3D, CorrectInputExpectCorrectOutput)
```

Comment:
Should this run for other make types besides only hydro?

Reply:
It should run for any make type that includes hydro, so the way the current builds are defined, that would include gravity and particles, but it doesn't have to, in principle (one could build with gravity and particles but no hydro, and this function would not get called).

Reply:
Ok, as is it will only run with the make type "hydro". If it should run with other make types then you should add them to the suite name

Reply:
I think because they could be redefined without the hydro dependence, it makes more sense to confine it to hydro.